src/mmu.rs

```diff
@@ -43,23 +43,23 @@ const HRAM_START: usize = 0xFF80;
 const HRAM_END: usize = 0xFFFE;
 const IE_REGISTER: usize = 0xFFFF;
 
-pub struct MMU<'a> {
-    mbc: &'a mut MBC0,
-    wram: WRAM,
-    hram: HRAM,
+pub struct MMU {
+    pub mbc: MBC0,
+    pub wram: WRAM,
+    pub hram: HRAM,
```

Comment:
Why are WRAM and HRAM public ? They are never used outside the MMU

Reply:
https://github.com/hferraud/gbmu/pull/7/commits/d94330803f50c0a1aae7cf469d93678028c6d5ff